observatory-platform/observatory/platform/terraform/terraform_builder.py

```diff
@@ -172,6 +172,27 @@ def build_terraform(self):
         self.make_files()
         self.platform_runner.make_files()
 
+    def install_packer_plugins(self) -> Tuple[str, str, int]:
+        """Install the necessary plugins for packer."""
+
+        # Install the Packer plugins by doing an init on the observatory image config file
+        args = ["packer", "init", "observatory-image.json.pkr.hcl"]
```

Comment:
Is the image config file always in the same directory that the packer script is being run? Would it be better to supply the absolute path?

Reply:
It is OK because the current working directory is set below `cwd=self.terraform_build_path`